### ISSUE TITLE
Does Putty Client Work Well with Cloudflare access if not when we can expect the feature?

### ISSUE THREAD

**Issue by deeepakkumarthakur16** [cloudflare/cloudflared#638]
**Does Putty Client Work Well with Cloudflare access if not when we can expect the feature?**

As putty is most used software for ssh for windows users. **Also how we can ssh for machine using cloudflare access from windows apart from putty if putty not in your list?**

**Comment by sudarshan-reddy** [comment]
@deeepakkumarthakur16 unsure what your expectation is.

Do you want Putty to be able to connect with cloudflared access?

**Comment by deeepakkumarthakur16** [comment]
Yes, I want putty should be able to connect with cloudflare access for ssh.
